### DISCUSSION TITLE
Wrong markdown highlighting colors

### DISCUSSION THREAD

**Discussion by kuraga** [community/community#70231] [Repositories]
### Select Topic Area

Bug

### Body

https://github.com/deeppavlov/DeepPavlov/blob/72bc642c2b51f1cc74dddc5b1b5d696e0badbdbb/CONTRIBUTING.md?plain=1

Markdown code highlighting colors are invalid (see screenshot).
All highlited lines are text. But they are highlighted as a code.

P.S. Which library does highlight this?

![image](https://github.com/community/community/assets/1063219/f5b3e966-aed6-4256-bddf-242541a76550)

**Comment by ramazansancar** [comment]
Hi @kuraga,
To see the codes in plain color, you need to write your code inside 3 ` (back quotes) and specify the language in the first line.

See here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting

**Comment by kuraga** [comment]
@ramazansancar , thanks but why?

**Reply by ramazansancar** [reply]
It's structured that way. :/

**Reply by kuraga** [reply]
@ramazansancar I mean it's a bug report. I don't intend to see plain colors.

Maybe I should make precise.
All highlited lines are text. But they are highlighted as a code.
